### ISSUE TITLE
Docblock interpreted as File-level instead of Class-level with readonly class

### ISSUE THREAD

**Issue by benjaminciottisonergia** [PHPCSStandards/PHP_CodeSniffer#283]
## Describe the bug
A class docblock is interpreted as a file docblock depending on how the class is declared (final, readonly)

phpcs version used : 3.7.2

## # Way to reproduce
```
<?php

namespace MyNameSpace;

use MyOtherClass;

/**
 * blabla
 */
class MyClass
{
```
is correctly interpreted as class docblock

```
<?php

namespace MyNameSpace;

use MyOtherClass;

/**
 * blabla
 */
final readonly class MyClass
{
```
is correctly interpreted as class docblock

```
<?php

namespace MyNameSpace;

use MyOtherClass;

/**
 * blabla
 */
readonly class MyClass
{
```
is wrongly interpreted as file docblock

**Comment by jrfnl** [comment]
@benjaminciottisonergia Closing as invalid.

Please use an up to date version of PHPCS. The latest version is 3.8.1.

Also please follow the bug report template properly as it is now completely unclear what sniffs are involved in this and what the problem is you are reporting.

I have presumed the problem was with the `PEAR.Commenting.ClassComment` sniff or the `Squiz.Commenting.ClassComment` sniff and in both cases, this was already fixed in PHPCS 3.8.0.
